<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Generator</class>
 <widget class="QWidget" name="Generator">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>521</width>
    <height>351</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="0">
    <widget class="QLabel" name="LblGeneratedStudentNumber">
     <property name="font">
      <font>
       <pointsize>60</pointsize>
      </font>
     </property>
     <property name="text">
      <string>-</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="2">
    <widget class="QPushButton" name="PbGenerate">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>25</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>15</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Генерировать</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLabel" name="LblTaskHeader">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>100</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>50</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Задача №:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="LblSudentHeader">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>100</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>50</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Ученик №:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLabel" name="LblGeneratedTaskNumber">
     <property name="font">
      <font>
       <pointsize>60</pointsize>
      </font>
     </property>
     <property name="text">
      <string>-</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
